packages/maskbook/src/plugins/ITO/UI/CreateForm.tsx

```diff
@@ -291,26 +322,24 @@ export function CreateForm(props: CreateFormProps) {
                 {StartTime} {EndTime}
             </Box>
             {account ? (
-                <ActionButton
-                    onClick={async () => {
-                        const contract = createContract<Qualification>(
-                            account,
-                            // '0x12d13b8a15368087c8c1fe9f9670c4c5c93387aa', // good qualification contract ropsten
-                            '0x36b2b0A09d5c77d705F21ECF4e00390005c55D09', // bad qualification contract ropsten
-                            QualificationABI as AbiItem[],
-                        )
-
-                        if (!contract) return
-
-                        const isERC721 = await contract.methods.supportsInterface('0x01ffc9a7').call({ from: account })
-                        const isQualification = await contract.methods
-                            .supportsInterface('0xa497ab4b')
-                            .call({ from: account })
-
-                        console.log({ isERC721, isQualification })
-                    }}>
-                    check
-                </ActionButton>
+                <Box className={classes.line}>
+                    <TextField
+                        className={classes.input}
+                        label={t('plugin_ito_qualification_label')}
+                        onChange={qualificationChange}
+                        value={qualification}
+                        InputLabelProps={{
+                            shrink: true,
+                        }}
+                        InputProps={{
+                            endAdornment: verify ? (
+                                <Box className={classes.iconWrapper}>
+                                    <CheckIcon style={{ color: '#77E0B5', height: 16, width: 16 }} />
```

Comment:
Perhaps it would be better to use the `size`
https://next.material-ui.com/components/icons/#size

Reply:
done